GeoscanDecoder/ui.py

```diff
@@ -139,6 +139,8 @@ def __init__(self, config):
         self.update()
         self.master.minsize(self.winfo_width(), self.winfo_height())
 
+        self.is_about_created = False
+
```

Comment:
this is unnecessary because the TopLevel 's About window does not allow the focus to be shifted to the parent window where F1 is handled

Reply:
![image](https://github.com/baskiton/GeoscanDecoder/assets/60815271/cf7c43a2-5109-421d-a604-63abcfb3c803)

Reply:
understood, reproduced
just add `about.focus_set()`
``` diff
 about.transient(self)
+about.focus_set()
 about.grab_set()
```